cloud-gateway-service/src/test/java/org/zowe/apiml/cloudgatewayservice/acceptance/xForwardHeaders/MutateRemoteAddressFilter.java

```diff
@@ -20,13 +20,19 @@
 import org.springframework.web.server.WebFilter;
 
 import java.net.InetSocketAddress;
+import java.util.concurrent.atomic.AtomicReference;
 
 @Configuration
 @Profile("forward-headers-proxy-test")
 public class MutateRemoteAddressFilter {
 
-    @Value("${test.proxyAddress}")
-    public String proxyAddress;
+    public AtomicReference<String> proxyAddressReference;
+    private String originalProxyAddressProperty;
```

Comment:
Could it be `final`?
```suggestion
    private final String originalProxyAddressProperty;
```

Reply:
done